src/commands/workflow/get.ts

```diff
@@ -29,17 +44,49 @@ export default class WorkflowGet extends BaseCommand<typeof WorkflowGet> {
   static enableJsonFlag = true;
 
   async run(): Promise<ApiV1.GetWorkflowResp | void> {
-    const resp = await withSpinner<ApiV1.GetWorkflowResp>(() =>
-      this.apiV1.getWorkflow(this.props),
-    );
+    spinner.start("‣ Loading");
+
+    const { workflow, whoami } = await this.loadWorkflow();
+
+    spinner.stop();
+
+    if (!workflow || !whoami) return;
 
     const { flags } = this.props;
-    if (flags.json) return resp.data;
+    if (flags.json) return workflow;
+
+    this.render(workflow, whoami);
+  }
+
+  private async loadWorkflow(): Promise<{
+    workflow?: ApiV1.GetWorkflowResp;
+    whoami?: ApiV1.WhoamiResp;
+  }> {
+    const workflowResp = await this.apiV1.getWorkflow(this.props);
+
+    if (!isSuccessResp(workflowResp)) {
+      const message = formatErrorRespMessage(workflowResp);
+      ux.error(new ApiError(message));
+
+      return {};
```

Comment:
Fyi `ux.error()` will basically throw and result in aborting out of the command, so i don't think we can ever get to this `return`